R/derive_param_tte.R

```diff
@@ -322,7 +322,11 @@ derive_param_tte <- function(dataset = NULL,
                              censor_conditions,
                              create_datetime = FALSE,
                              set_values_to,
-                             subject_keys = get_admiral_option("subject_keys")) {
+                             subject_keys = get_admiral_option("subject_keys"),
+                             check_type = "warning") {
+  # Match check_type to valid admiral options
+  check_type <- rlang::arg_match(check_type, c("warning", "message", "error", "none"))
```

Comment:
Please use the admiraldev assertions (`assert_character_scalar()`) and move it to the "checking and quoting" section below.